lib/installedtest.pm

```diff
@@ -112,8 +112,10 @@ sub collect_data {
     upload_logs('/var/tmp/var_log.tar.gz');
 
     # Run eos-diagnostics and upload its results
-    assert_script_run('eos-diagnostics /var/tmp/eos-diagnostics.txt');
-    upload_logs('/var/tmp/eos-diagnostics.txt');
+    # Older versions of it can fail if $DISPLAY isn’t set
```

Comment:
Do we ship `xvfb-run` in the ostree? Though It's probably not worth doing a more elaborate workaround given you've fixed eos-diagnostics

Reply:
I don’t think it’s worth doing the more-elaborate workaround, since this code is mainly used to diagnose test failures — which we will typically see on nightly images which already contain the `eos-diagnostics` fix. It might be useful for diagnosing test failures on newly-tested old releases (e.g. when testing the 3.3 → 3.4 upgrade path) but I’m happy to burn that bridge when we come to it.